px-auto-filter-field.html

```diff
@@ -0,0 +1,139 @@
+<link rel="import" href="../polymer/polymer-element.html"/>
+<link rel="import" href="px-data-grid-theme.html">
+<link rel="import" href="../app-localize-behavior/app-localize-behavior.html"/>
+
+<dom-module id="px-auto-filter-field">
+  <template>
+    <style>
+
+      :host {
+        position: relative;
+        display: block;
+      }
+
+      .field {
+        background-color: var(--px-auto-filter-field-background-color, red);
```

Comment:
you can exclude some stylelint rules here: https://github.com/vaadin/px-data-grid/blob/master/.stylelintrc

Reply:
To be fixed later.